app/components/request_metrics/request_metrics.rb

```diff
@@ -21,7 +21,7 @@ def metrics
       [
         {
           value: stats[:counts][:contributors],
-          total: stats[:counts][:recipients],
+          total: request_for_info.organization.contributors.active.with_tags(request_for_info.tag_list).count,
```

Comment:
This change became necessary because relying on the count of created outbound messages is flaky, especially when scheduling a job that creates them. What this meant in practice is that the user was redirected to the request show page after the request was successfully created, but since the message records themselves had not been created it falsely reported that no one had received the message.